### ISSUE TITLE
Criar tela de participantes 

### ISSUE THREAD

**Issue by Viniciuscarvalho** [CocoaHeadsBrasil/CocoaHeadsApp#96]

**Comment by barbosa** [comment]
![lista de presenca](https://cloud.githubusercontent.com/assets/235208/16419210/e32388fe-3d1a-11e6-8876-086a8556b010.png)

**Comment by thiagoramos23** [comment]
E aí galera, alguém pegou essa tela pra fazer?

**Comment by Viniciuscarvalho** [comment]
Ainda não @thiagoramos23 , mas ela é tem um pré requisito que é a tela de eventos, que não foi feita ainda.

**Comment by barbosa** [comment]
@Viniciuscarvalho vc estava fazendo a listagem de eventos?

**Comment by Viniciuscarvalho** [comment]
Sim, vou fazer um commit com minhas alterações e já faço PR @barbosa

**Comment by thiagoramos23** [comment]
@Viniciuscarvalho vou pegar essa então. Como só consigo me envolver com open source a noite eu creio que não seja problema. Ok?

**Comment by Viniciuscarvalho** [comment]
Tudo certo @thiagoramos23 , farei o commit agora a tarde e se tiver tudo certo no PR, você já pode ver essa lista a noite.

**Comment by barbosa** [comment]
Imagina, @thiagoramos23. Sem problemas.

Qualquer dúvida que tiver, só perguntar 👍

**Comment by thiagoramos23** [comment]
@Viniciuscarvalho poderia dar um assign dessa feature pra mim?

**Comment by Viniciuscarvalho** [comment]
não é possível você dar assign yourself? @thiagoramos23

**Comment by barbosa** [comment]
É pq vc não estava como colaborador do projeto. Acabei de te adicionar, @thiagoramos23 

Confirma o invite que vc deve ter recebido no seu email. Depois, é só voltar aqui e dar assign pra vc.
